cirq-web/example.ipynb

```diff
@@ -0,0 +1,177 @@
map,Vs.fog,Vs.lights,{emissive:{value:new Dr(0)},specular:{value:new Dr(1118481)},shininess:{value:30}}]),vertexShader:Us.meshphong_vert,fragmentShader:Us.meshphong_frag},standard:{uniforms:Ts([Vs.common,Vs.envmap,Vs.aomap,Vs.lightmap,Vs.emissivemap,Vs.bumpmap,Vs.normalmap,Vs.displacementmap,Vs.roughnessmap,Vs.metalnessmap,Vs.fog,Vs.lights,{emissive:{value:new Dr(0)},roughness:{value:1},metalness:{value:0},envMapIntensity:{value:1}}]),vertexShader:Us.meshphysical_vert,fragmentShader:Us.meshphysical_frag},toon:{uniforms:Ts([Vs.common,Vs.aomap,Vs.lightmap,Vs.emissivemap,Vs.bumpmap,Vs.normalmap,Vs.displacementmap,Vs.gradientmap,Vs.fog,Vs.lights,{emissive:{value:new Dr(0)}}]),vertexShader:Us.meshtoon_vert,fragmentShader:Us.meshtoon_frag},matcap:{uniforms:Ts([Vs.common,Vs.bumpmap,Vs.normalmap,Vs.displacementmap,Vs.fog,{matcap:{value:null}}]),vertexShader:Us.meshmatcap_vert,fragmentShader:Us.meshmatcap_frag},points:{uniforms:Ts([Vs.points,Vs.fog]),vertexShader:Us.points_vert,fragmentShader:Us.points_frag},dashed:{uniforms:Ts([Vs.common,Vs.fog,{scale:{value:1},dashSize:{value:1},totalSize:{value:2}}]),vertexShader:Us.linedashed_vert,fragmentShader:Us.linedashed_frag},depth:{uniforms:Ts([Vs.common,Vs.displacementmap]),vertexShader:Us.depth_vert,fragmentShader:Us.depth_frag},normal:{uniforms:Ts([Vs.common,Vs.bumpmap,Vs.normalmap,Vs.displacementmap,{opacity:{value:1}}]),vertexShader:Us.normal_vert,fragmentShader:Us.normal_frag},sprite:{uniforms:Ts([Vs.sprite,Vs.fog]),vertexShader:Us.sprite_vert,fragmentShader:Us.sprite_frag},background:{uniforms:{uvTransform:{value:new Jn},t2D:{value:null}},vertexShader:Us.background_vert,fragmentShader:Us.background_frag},cube:{uniforms:Ts([Vs.envmap,{opacity:{value:1}}]),vertexShader:Us.cube_vert,fragmentShader:Us.cube_frag},equirect:{uniforms:{tEquirect:{value:null}},vertexShader:Us.equirect_vert,fragmentShader:Us.equirect_frag},distanceRGBA:{uniforms:Ts([Vs.common,Vs.displacementmap,{referencePosition:{value:new ai},nearDistance:{value:1},farDistance:{value:1e3}}]),vertexShader:Us.distanceRGBA_vert,fragmentShader:Us.distanceRGBA_frag},shadow:{uniforms:Ts([Vs.lights,Vs.fog,{color:{value:new Dr(0)},opacity:{value:1}}]),vertexShader:Us.shadow_vert,fragmentShader:Us.shadow_frag}};function Ws(t,e,n,i,r){const s=new Dr(0);let a,o,l=0,c=null,h=0,u=null;function d(t,e){n.buffers.color.setClear(t.r,t.g,t.b,e,r)}return{getClearColor:function(){return s},setClearColor:function(t,e=1){s.set(t),l=e,d(s,l)},getClearAlpha:function(){return l},setClearAlpha:function(t){l=t,d(s,l)},render:function(n,r,p,g){let v=!0===r.isScene?r.background:null;v&&v.isTexture&&(v=e.get(v));const y=t.xr,x=y.getSession&&y.getSession();x&&\"additive\"===x.environmentBlendMode&&(v=null),null===v?d(s,l):v&&v.isColor&&(d(v,1),g=!0),(t.autoClear||g)&&t.clear(t.autoClearColor,t.autoClearDepth,t.autoClearStencil),v&&(v.isCubeTexture||v.mapping===ct)?(void 0===o&&(o=new bs(new Ms(1,1,1),new As({name:\"BackgroundCubeMaterial\",uniforms:Ss(ks.cube.uniforms),vertex
```

Comment:
[No change needed] I wasn't able to test this in colab due to `cirq_web` being unreleased. If you'd like, I can try getting this running locally to inspect more closely.

Reply:
You can test with `jupyter notebook` if you wanna try the Jupyter notebook, also have this Colab with the PyPI package if you want to try that as well: https://colab.research.google.com/drive/1Jukad4xTwU_f9YiJ8GsPiK6BidQQSm3s?resourcekey=0-9zSsrwsyeJIxq0SV4ccqWA&usp=sharing

Reply:
Thanks for the link! All seems good to me there.